src/main/groovy/io/cify/runner/utils/CapabilityParser.groovy

```diff
@@ -152,8 +236,8 @@ class CapabilityParser {
      *
      * @param text - capabilities file content in string format
      * @return LazyMap with capabilities
-     * */
-    static Object readFromContent(String text) {
+     */
+    private static Object readFromContent(String text) {
         try {
             JsonSlurper jsonParser = new JsonSlurper()
             def content = jsonParser.parseText(text)
```

Comment:
I think it's better to return an object anyway either NULL or Not Null instead of throwing an exception. You can handle an exception while parsing text though but pass the result anyway outside of try/catch.

Reply:
This will need additional testing. Let's make it as a separate task